virtesk-tc-connectspice/test_find_thinclient_identifier_nmcli.py

```diff
@@ -0,0 +1,48 @@
+import find_thinclient_identifier_nmcli as find_nmcli
+
+
+def test_extract_identifiers_from_nmcli(mock, nmcli_device_show, nmcli_con_show, nmcli_con_show_empty):
+    def my_check_output(cmd, **kwargs):
+        if cmd[:4] == ['nmcli', 'con', 'show', 'ADSY-EAP']:
+            return nmcli_con_show
+        elif cmd[:3] == ['nmcli', 'con', 'show']:
+            return nmcli_con_show_empty
+        elif cmd[:3] == ['nmcli', 'device', 'show']:
+            return nmcli_device_show
+    mock.patch('subprocess.check_output', my_check_output)
```

Comment:
I think this should also be find_thinclient_identifier_nmcli.subprocess.check_output